lib/node_modules/@stdlib/math/base/special/acoth/README.md

```diff
@@ -0,0 +1,87 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# acoth
+
+> Compute the [inverse hyperbolic cotangent][inverse-hyperbolic-cotangent] of a number.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var acoth = require( '@stdlib/math/base/special/acoth' );
+```
+
+#### acoth( x )
+
+Computes the [inverse hyperbolic cotangent][inverse-hyperbolic-cotangent] of a `number` (in radians).
+
+```javascript
+var v = acoth( 1.0 );
+// returns 0
+
+v = acoth( 2.0 );
+// returns ~0.5493
+
+v = acoth( NaN );
+// returns NaN
+```
+
+The domain of `x` is restricted to `[x < -1 || x > 1)`. If `x >= -1 || <= 1`, the function will return `NaN`.
```

Comment:
On a second look, if `x` is exactly -1 or 1, this implementation it will return 0. NaN will be returned if `x` is less than 1 and more than -1 but not equal. This was my mistake.